web/themes/custom/bixal_uswds/php-includes/paragraphs.inc

```diff
@@ -71,12 +71,14 @@ function bixal_uswds_preprocess_paragraph(&$vars) {
         $entity_query = \Drupal::entityQuery('node');
         $entity_query->condition('type', 'bixaler')
           ->accessCheck(TRUE)
+          ->condition('status', NodeInterface::PUBLISHED)
```

Comment:
Good call, even though we have access check set to true, the values would be OK for anon but admin would see un-pub as well with out this. Very interesting. Would be interesting to think about this issue if there was something besides published status that access check took into account.